.github/workflows/publish.yml

```diff
@@ -28,13 +28,13 @@ jobs:
 
       - name: Publish
         run: |-
-          sbt ci-release
-          CI_RELEASE=akka-grpc-codegen/publishSigned CI_SNAPSHOT_RELEASE=akka-grpc-codegen/publish sbt ++2.13.10\! ci-release
+          sbt +publishSigned
+          CI_RELEASE=akka-grpc-codegen/publishSigned CI_SNAPSHOT_RELEASE=akka-grpc-codegen/publish sbt ++2.13.10\! publishSigned
         env:
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          PUBLISH_USER: ${{ secrets.PUBLISH_USER }}
+          PUBLISH_PASSWORD: ${{ secrets.PUBLISH_PASSWORD }}
 
   gradle-plugin:
     name: Release gradle plugin
```

Comment:
I didn't try to change the gradle plugin publishing, might not be important and I have no clue

Reply:
My (limited) understanding is that the gradle plugins must come from their gradle plugin repo. Gradle users will still need the runtime and other jars from the new repo for it to work though (covered by the changes in walkthrough.md)